src/App.tsx

```diff
@@ -16,8 +17,41 @@ import httpClient from "./httpClient";
 import { setLanguageType } from "./utils/translate/translateService";
 
 function App() {
-  const [{ language }, dispatch] = useContext(AppContext);
-  // DATA
+  const [{ language, dataPageState, filters }, dispatch] = useContext(AppContext);
+  const history = useHistory();
+  setLanguageType(language);
+
+  useEffect(() => {
+    const toggleFilters: LocationListener = ({ pathname }: Location): void => {
+      if (pathname === "/Analyze") {
+        dispatch({
+          type: 'SELECT_EXPLORE_OR_ANALYZE',
+          payload: false
+        })
+      }
+      else if (pathname === "/Explore") {
+        dispatch({
+          type: 'SELECT_EXPLORE_OR_ANALYZE',
+          payload: true
+        })
+      }
+    }
+
+    const api = new httpClient()
```

Comment:
Make comment that this will be updated to use new endpoint to get filter options